ga4gh/datamodel/variants.py

```diff
@@ -238,6 +244,35 @@ def getVariants(self, referenceName, startPosition, endPosition,
                 else:
                     break
 
+    def getCallSets(self, name, startCallSetPosition):
+        """
+        Returns an iterator over the specified callset name. The parameters
+        correspond to the attributes of a GASearchCallSetsReuqest object.
+        """
+        assert name is not None
```

Comment:
Take out this assert and throw an exception here instead.  You can just create an Exception("your msg here") object with a TODO comment that says to use a better-thought-out exception class later.